### PR TITLE
Updates Dela+Go and fixes the tests

### PR DESCRIPTION
**Pull request by nkcr**
- Uses the latest version of Dela
- Fixes the tests (encoding of ballot ID, warnings instead of errors)
- Uses Go 1.19
- Cleans some parts (removes prints, formats lines)

### CODE REVIEW

File: contracts/evoting/types/ballots_test.go [hunk 1]
```diff
@@ -382,14 +383,6 @@ func TestSubject_IsValid(t *testing.T) {
...
 	require.True(t, valid)
 
-	// with wrongly ID not in base64
-	mainSubject.ID = "zzz"
-
-	configuration.Scaffold = []Subject{*mainSubject}
-
-	valid = configuration.IsValid()
-	require.False(t, valid)
-
```
Comment (emduc) [inline contracts/evoting/types/ballots_test.go:392]: Not sure why the test is removed, do we not enforce base64 IDs anymore?

**Review by emduc — APPROVED** [review]
Seems all good to me, thanks for the request! :)
It's much clearer with warnings instead of errors indeed.

File: contracts/evoting/types/ballots_test.go (hunk 1 above)
Comment (nkcr) [reply contracts/evoting/types/ballots_test.go:392]: Not sure anymore. Isn't it something you changed based on some errors we had on the frontend ?

Comment (emduc) [reply contracts/evoting/types/ballots_test.go:392]: You're right, we no longer require the IDs to be base64 encoded in the configuration, but only when the ballots are cast

**Comment by nkcr** [comment]
Wow, all tests passing. Haven't see that for a while 😄

**Pull request merged**